internal/sftp/goclient_test.go

```diff
@@ -287,4 +286,60 @@ func TestGoClient(t *testing.T) {
 			assert.Assert(t, tfs.Equal(dest.Path(), tfs.Expected(t, tc.want.Paths...)))
 		})
 	}
+
+	type deleteTest struct {
```

Comment:
I think it's better to move this to it's own test, e.g.

```
func TestDelete(t *testing.T) {
```

And rename the first test `TestUpload` (or similar).

Reply:
Yes I was considering this. I will go through with this.